### PR TITLE
chore: bump deps

### PR DESCRIPTION
**Pull request by frezbo**
Bump deps

Signed-off-by: Noel Georgi <git@frezbo.dev>

### CODE REVIEW

**Comment by frezbo** [comment]
kernel bump also fixes: CVE-2023-0179

File: Pkgfile [hunk 1]
```diff
@@ -58,19 +58,19 @@ vars:
...
-  ipxe_sha512: d9123ec10746c6e0a94ec6c68002a7c73e0cac5ab513cf956288dd84853b9e4aa8262fb7fd28fd30383b06bb0c8a6b49abea653d779fa92eba193eca73abcf6c
+  ipxe_ref: 08740220baba87cbc6acb1c00cd5b492ac0c5a08
+  ipxe_sha256: 805a9ff31c1ae1144ff7607e0f1a4ac1ecad41b50e8f9fb079caad52cfbdb46a
+  ipxe_sha512: 4fd9ff063b143bf96b41749bc3c01d5ed7ad162325639bc5df974f0bbd2cd3eb2ac67eee60a02b9bd13ab58589c6a5a4b15500abfe86f1dcad1339737bd657e5
 
   # renovate: datasource=git-tags extractVersion=^v(?<version>.*)$ depName=git://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git
-  linux_version: 6.1.5
-  linux_sha256: bc7f6d9a8a8bbe9a723e82346bba94b58d926f78bfba106b21e041e0290076fc
-  linux_sha512: f359a8ff3d2d4f1e95bd90431209fa17b7b37029a246b4fbadd618afb734f2e0385a21ebdc21c9fe6b5cc84876b34a59d7ae582c97c34cf29238231db4161275
+  linux_version: 6.1.7
```
Comment (smira) [inline Pkgfile:71]: `make kernel-olddefconfig` ?

Comment (frezbo) [reply Pkgfile:71]: forgot that :sweat_smile:

**Review by smira — APPROVED** [review]

**Pull request merged**